src/actions.js

```diff
@@ -1,64 +1,69 @@
 import { fetchCocktailEnrichment } from "./services/cocktailDBAPI.service";
 
+import { LOAD_COCKTAILS, LOAD_INGREDIENTS, LOAD_GLASSES, UPDATE_FILTER,
```

Comment:
Would it be okay to include these as;
```
import * as actionTypes from './actionTypes';
```
and then refer to them like;
```
return { type: actionTypes.LOAD_COCKTAILS, payload };
```
Same in the other file - this big import at the top isn't going to scale and it means we'll need to change code in two places every time a new action type comes along.

Reply:
You're right! I'll work on it tomorrow. Thanks for the feedback.